packages/lib/factorio/patch.js

```diff
@@ -152,7 +152,7 @@ function reorderDependencies(modules) {
 
 		// No unmet dependencies, record and continue
 		present.set(module.name, module.version);
-		index++;
+		index += 1;
```

Comment:
Is there like a lint rule or something that mandate changing it from increment to this form?

Reply:
There was a warn for it, he made it an error instead. See .eslintrc.js https://github.com/clusterio/factorioClusterio/pull/337/commits/5337280eec4426873e11e6aedbc6602ecc904a94